package-lock.json

```diff
@@ -1,21 +1,17 @@
 {
     "name": "vanilla-token-auth",
-    "version": "1.0.3",
+    "version": "1.0.4",
```

Comment:
Wups, I must have forgotten to `npm i` when I bumped the version to `1.0.4`. I wonder what that means for the `npm publish` I did? Oh well, we'll bump again after this merges anyway.

Reply:
NPM doesn't care about the `package-lock.json` file. It only cares about the package.json and the remaining files that are packed into it.

You can check what files are being packed by running `npm pack`.